frontend/public/components/network-policies/network-policy-form.tsx

```diff
@@ -0,0 +1,390 @@
+        </label>
+        <input
+          className="pf-c-form-control"
+          type="text"
+          onChange={handleNameChange}
+          value={networkPolicy.name}
+          placeholder="my-policy"
+          id="name"
+          name="name"
+          required
+        />
+      </div>
+      <div className="form-group co-create-networkpolicy__namespace">
+        <label className="co-required" htmlFor="namespace">
+          {t('public~Policy namespace')}
+        </label>
+        <input
+          className="pf-c-form-control"
+          type="text"
+          onChange={handleNamespaceChange}
+          value={networkPolicy.namespace}
+          id="namespace"
+          name="namespace"
+          aria-describedby="namespace-help"
+          required
+        />
+        <div className="help-block" id="namespace-help">
+          <p>{t('public~The namespace containing pods on which this network policy applies.')}</p>
+        </div>
+      </div>
+      <div className="form-group co-create-networkpolicy__podselector">
+        <span>
+          <label>{t('public~Pod selector')}</label>
+        </span>
+        <div className="help-block" id="pod-selector-help">
+          <p>
+            {t(
+              'public~If no pod selector is provided, the policy will apply to all pods in the namespace. Else, the policy will apply to pods in the namespace with labels that include all the supplied key/value pairs.',
+            )}
+          </p>
+        </div>
+        {showPodSelector ? (
+          <NameValueEditorComponent
+            nameValuePairs={networkPolicy.podSelector}
+            valueString={t('public~Selector')}
+            nameString={t('public~Label')}
+            addString={t('public~Add label')}
+            readOnly={false}
+            allowSorting={false}
+            updateParentData={handleMainPodSelectorChange}
+            onLastItemRemoved={() => setShowPodSelector(false)}
+          />
+        ) : (
+          <>
+            <div className="co-toolbar__group co-toolbar__group--left co-create-networkpolicy__add-pod-selector">
+              <Button
+                className="pf-m-link--align-left"
+                onClick={() => setShowPodSelector(true)}
+                type="button"
+                variant="link"
+              >
+                <PlusCircleIcon className="co-icon-space-r" />
+                {t('public~Add pod selector')}
+              </Button>
+            </div>
+          </>
+        )}
+      </div>
+      <div className="form-group co-create-networkpolicy__type">
+        <Title headingLevel="h2">{t('public~Policy type')}</Title>
+      </div>
+      {showSDNAlert && (
+        <Alert
+          variant="info"
+          title={t(
+            'public~When using the OpenShift SDN cluster network provider, egress network policy is not supported.',
+          )}
+          actionClose={<AlertActionCloseButton onClose={() => setShowSDNAlert(false)} />}
+        />
+      )}
```

Comment:
Shall we move this alert to `Egress` rules when user tries to add `Egress` rules?

Reply:
<img width="946" alt="Screen Shot 2021-06-01 at 4 10 32 PM" src="https://user-images.githubusercontent.com/12692381/120289666-1d0afc00-c2f4-11eb-8111-40c3a44b161c.png">

Reply:
The first place where "egress" is mentioned, hence where this information is useful, is the checkbox "Deny all egress traffic", so that's why I think it's preferable to have this alert prior to it. The top of the "Policy type" section made sense to me.
For that reason I think we shouldn't remove it from there ; eventually we could duplicate it so that's it's also visible in Egress. The downside is that it adds more redundant information on screen, which some users could perceive as noise. So, I'm not well decided. @andrew-ronaldson , any thought about that?

Reply:
Or perhaps, we could have it displayed as a tooltip on the "deny all egress" checkbox + as this alert in the egress section. wdyt?